portfolio/src/main/webapp/script.js

```diff
@@ -56,7 +57,84 @@ function getRandomFile(element, path, fileName, numElems) {
  * @param {DOM element} element - an element in the DOM for selecting
  */
 function fetchAndModify(path, element) {
-  fetch(path).then(response => response.text()).then(content => 
-    element.innerHTML= content
-  )
+  fetch(path)
+      .then(response => response.text())
+      .then(content => element.innerHTML = content)
 }
+/*
+ * This code creates an innerHTML and then reads its text to remove special
+ * characters from the input
+ * @param {string} str The input to remove special characters from to prevent
+ *     scripting attacks
+ */
+var sanitizeHTML = function(str) {
```

Comment:
Don't use `var`: https://google.github.io/styleguide/jsguide.html#features-use-const-and-let

And, in this case, I think it's simpler to just write `function sanitizeHtml(html) { ... }` anyway.